modules/programs/alacritty.nix

```diff
@@ -9,6 +9,31 @@ in {
 
       package = lib.mkPackageOption pkgs "alacritty" { };
 
+      theme = lib.mkOption {
+        type = let
+          themes = lib.pipe pkgs.alacritty-theme [
+            builtins.readDir
```

Comment:
Are we confident in this? It sure seems like this is enumerating the wrong directory. Do we need to specifically enumerate the `/share/alacritty-theme` directory within the package?

Reply:
Actually, this just feels wrong to do anyway: in general we should not be importing the package into the closure for config. Let me see what I can do to resolve this...

Reply:
I'm not convinced that we should restrict this to just a theme from the `alacritty-themes` repository. I think that we should just allow the user to specify a file, and have as an example using the alacritty-theme package.

Reply:
Actually, there is an argument for having this, and I think that we should keep it but also note how to configure a theme that isn't in the alacritty-theme package.